### PR TITLE
Move admin section to workflow section

### PR DESCRIPTION
**Pull request by rullzer**
* Move to workflow admin section
* Add test
* Remove old settings registration
* Bump version

Fixes https://github.com/nextcloud/files_retention/issues/8

### CODE REVIEW

**Comment by rullzer** [comment]
CC @nickvergessen

**Comment by nickvergessen** [comment]
Works :+1:

File: lib/Settings/Admin.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace OCA\Files_Retention\Settings;
```
Comment (nickvergessen) [inline lib/Settings/Admin.php]: Missing license!

File: tests/lib/Settings/AdminTest.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace OCA\Files_Retention\Tests\Settings;
```
Comment (nickvergessen) [inline tests/lib/Settings/AdminTest.php]: same here

**Pull request merged**